MHArmory.Search/OpenCL/Host.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.ObjectModel;
+using System.Text;
+using Cloo;
+
+namespace MHArmory.Search.OpenCL
+{
+    class Host : IDisposable
```

Comment:
Please set modifier explicitly, such as `public class Host`. By default the class is `internal`, if you want to keep it internal, then please add the `internal` keyword to make it explicit.